src/DetailsView/reports/automated-checks-report.scss

```diff
@@ -250,3 +250,73 @@ $outcome-not-applicable-summary-color: $neutral-outcome;
 .not-applicable-checks-section {
     margin-top: 56px;
 }
+
+.a11y-report-instance-table {
+    table,
+    thead,
+    tbody,
+    th,
+    td,
+    tr {
+        display: block;
+    }
+
+    table {
+        font-size: 14px;
+        line-height: 20px;
+    }
+
+    thead tr {
+        position: absolute;
+        top: -9999px;
+        left: -9999px;
+    }
+
+    tr {
+        box-shadow: 0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132);
+        border-radius: 4px;
+        margin-bottom: 16px;
+        background: $neutral-0;
+    }
+
+    td {
+        border: none;
+        position: relative;
+        padding-left: 110px;
+        padding-top: 14px;
+        padding-right: 14px;
+        padding-bottom: 14px;
+
+        border-bottom: 0.5px solid $neutral-10;
+
+        color: $secondary-text;
+    }
+
+    td:nth-of-type(2) {
+        font-family: Consolas, Segoe UI, Courier New, monospace;
+    }
+
+    td:before {
+        position: absolute;
+        top: 14px;
+        left: 20px;
+        width: 66px;
+        padding-right: 10px;
+        white-space: nowrap;
+
+        font-family: $semiBoldFontFamily;
+        color: $primary-text;
+    }
+
+    td:nth-of-type(1):before {
+        content: 'Path';
```

Comment:
any idea, why we need this?

Reply:
If you look at `thead tr` style... we're displaying the headers off screen (but not using display:none so AT can read it anyway). The before selector on head `td` let us show the header (visually) and repeat it on each "card" without breaking the table semantics